cf/test/test_Data.py

```diff
@@ -3927,6 +3927,9 @@ def test_Data_set_units(self):
         with self.assertRaises(ValueError):
             d.set_units("km")
 
+    def test_Data_get_filenames(self):
```

Comment:
Oh dear - quite right - I'd forgotten about that! I've added a skip, and won't merge this yet.

Reply:
Thanks.

> and won't merge this yet.

Oh, sorry if I wasn't clear, I meant it is fine to merge so long as the test is skipped or otherwise made to not fail. So we're now good to merge I think - I'll do an approval now to formally confirm that!